doc/TUTORIALS_CN.md

```diff
@@ -16,6 +16,8 @@
     - [模型加载与预测](#模型加载与预测)
   - [逻辑回归](#逻辑回归)
   - [支持超大数据集](#支持超大数据集)
+- [隐私深度学习](#隐私深度学习)
```

Comment:
how about also adding in the English version TUTORIALS.md doc?

Reply:
ok, i will do it.